mule-gateway/modules/ROOT/pages/mule-gateway-proxy-config.adoc

```diff
@@ -0,0 +1,95 @@
+= Connect the Mule Gateway Through a Proxy Server
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can configure the Mule Gateway to route traffic through the proxy server to API Manager if your Mule Runtime engine runs inside a firewall that restricts external communication through a proxy server.
+
+Mule Gateway supports Basic, NTLM, and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
+
+
+== Configure the Mule Gateway to Connect Through a Proxy Server
+
+To configure Mule Gateway to connect through a proxy server, you must set up your proxy server configuration in a `$MULE_HOME/conf/wrapper.conf` file:
```

Comment:
Why? Although it might sound obvious, Kerberos requires extra config files, so making it clear that these properties are for the wrapper.conf seems ok to me